tests/test_requirements.txt

```diff
@@ -12,7 +12,7 @@ nose==1.3.7
 pandas==0.25.3
 pytest~=5.2
 requests==2.20.0
-scipy==1.3.3
+scipy==1.3.2
```

Comment:
This is the closest version that has a build for Windows on conda-forge. If this causes issues, this can either be upgraded or downgraded.

Reply:
maybe specify `~=1.3` then ? I don't think we should ping patch versions any way.

Reply:
We are pinning patch versions elsewhere in this file. I think it's all right for the test setup to pin like this.